server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/api/MailQueueView.java

```diff
@@ -34,6 +34,10 @@
 
     CompletableFuture<Void> deleteMail(Mail mail);
 
+    CompletableFuture<Long> delete(DeleteCondition deleteCondition);
```

Comment:
deleteMail is a special case of delete with a DeleteCondition that would match a single mail, right ?